### PR TITLE
Classref: Added PacketPeer*

### PR DESCRIPTION
**Pull request by J08nY**
Synced, fixed a wrong return type in `PacketPeer.get_var()` docs, and did the classref for `PacketPeer*`

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -15835,13 +15841,33 @@ Example: (content-length:12), (Content-Type:application/json; charset=UTF-8)
 			<argument index="0" name="device" type="int">
 			</argument>
 			<description>
-			Returns the duration of the current vibration effect in seconds.
+			If the device has an accelerometer, this will return the movement.
```
Comment (akien-mga) [inline doc/base/classes.xml]: Looks like something wrong happened here.

Comment (J08nY) [inline doc/base/classes.xml]: Hmm, I see. But this commit is literally just me running doctool on a freshly build godot. Will fix it though.

Comment (akien-mga) [inline doc/base/classes.xml]: That's interesting. There's likely a bug in doctool then. @bojidar-bg ? :D

**Comment by J08nY** [comment]
Fixed the sync.

**Pull request merged**